Makefile

```
@@ -0,0 +1,250 @@
+EXAMPLE_JAVA := example-java
+EXAMPLE_KOTLIN := example-kotlin
+EXAMPLE_SPRING := example-spring
+
+# Logs Directory
+LOG_DIR := build/logs
+
+# ------------------------------------------------------------------------------------
+# CORE & HELP
+# ------------------------------------------------------------------------------------
+
+help: ## Show this help message
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (14).

(maxbodylength)